masterlist.yaml

```diff
@@ -22679,6 +22679,9 @@ plugins:
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
       - 'Immersive Citizens - OCS patch.esp'
+    req: [ *SKSE ]
     clean:
       - crc: 0xBD12E363
         util: 'SSEEdit v4.0.4'
```

Comment:
We usually remove cleaning data for old versions of a mod.

```suggestion
```

Reply:
Thanks. I was unsure as it's not consistent in the file.